cucumber-expressions/go/examples.txt

```diff
@@ -41,3 +41,7 @@ Some $3.50 of cukes at 42° Celsius
 I select the {int}st/nd/rd/th Cucumber
 I select the 3rd Cucumber
 [3]
+---
+/^I am agent ([ABJ])$/
+I am agent J
+["J"]
```

Comment:
This is a JSON String, so no conversion to char would have taken place.

Reply:
Ok, then this test seems to be futile. I can get rid of it.